src/java.base/unix/native/libjava/ProcessImpl_md.c

```diff
@@ -773,4 +773,3 @@ Java_java_lang_ProcessImpl_forkAndExec(JNIEnv *env,
     closeSafely(err[0]); err[0] = -1;
     goto Finally;
 }
-
```

Comment:
Unintended change?

Reply:
IntelliJ removed a blank line at the end of the file.  I think it's ok to make permanent (in this PR).